itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenarios.java

```diff
@@ -1559,6 +1645,76 @@ public void testIncrementalLoad() throws IOException {
     verifyRun("SELECT a from " + replDbName + ".ptned WHERE b=2", ptnData2, driverMirror);
   }
 
+  @Test
+  public void testIncrementalLoadLazyCopy() throws IOException {
```

Comment:
Is this test in different mini hdfs set up?

Reply:
no, and I don't think that should be a concern here?

Reply:
You can add this test in the mini hdfs set up to make sure data copy is happening correctly in case of lazy load false and true.

Reply:
There are many existing tests with lazy load false. For external table which uses mini hdfs we already have a test for lazy load true. I will add one for other table as well.